contract_manager/src/contracts/near.ts

```diff
@@ -108,21 +113,38 @@ export class NearPriceFeedContract extends PriceFeedContract {
     return {
       chain: this.chain.getId(),
       address: this.address,
+      governanceDataSourceChain: this.governanceDataSource.emitterChain,
+      governanceDataSourceAddress: this.governanceDataSource.emitterAddress,
+      lastExecutedGovernanceSequence: this.lastExecutedGovernanceSequence,
       type: NearPriceFeedContract.type,
     };
   }
 
   static fromJson(
     chain: Chain,
-    parsed: { type: string; address: string }
+    parsed: {
+      type: string;
+      address: string;
+      governanceDataSourceAddress: string;
+      governanceDataSourceChain: number;
+      lastExecutedGovernanceSequence: number;
```

Comment:
I wonder where you are using these 🤔

Reply:
They are used in the corresponding methods of `NearPriceFeedContract`.